.github/workflows/release.yml

```diff
@@ -16,23 +25,44 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: '16'
+          cache: 'yarn'
       - name: Setup Git
         run: |
           git config user.name "GitHub Bot"
           git config user.email "gituser@example.com"
-      - uses: actions/setup-node@v3
-        with:
-          node-version: '14'
-          cache: 'yarn'
       - name: Install packages
         run: yarn install --frozen-lockfile
+      - name: Lint
+        shell: bash
+        run: |
+          PROJECTS=$($(yarn bin)/nx print-affected --type=lib --select=projects --base=origin/main)
+          if [[ "$PROJECTS" ]]; then
+            yarn nx run-many --target=lint --projects="$PROJECTS"
+          else
+            echo "No changes, Skipping lint check."
+          fi
+      - name: Build
+        shell: bash
+        run: |
+          PROJECTS=$($(yarn bin)/nx print-affected --type=lib --select=projects --base=origin/main)
+          if [[ "$PROJECTS" ]]; then
+            yarn nx run-many --target=build --projects="$PROJECTS"
+          else
+            echo "No changes, Skipping lint check."
```

Comment:
Please change the message to be relevant for build step

Reply:
Done